### ISSUE TITLE
"Say Cheese" shouldn't be a string?

### ISSUE THREAD

**Issue by DanFloodTechnologies** [googlesamples/io2015-codelabs#100]
In the example code the line is: 

getActivity().getVoiceInteractor()
                .submitRequest(new PickOptionRequest(
                        "Say Cheese",
                        new Option[]{option},
                        null) {

But it seems that it should be:

getActivity().getVoiceInteractor()
                .submitRequest(new PickOptionRequest(
                        new VoiceInteractor.Prompt("Say Cheese"),
                        new Option[]{option},
                        null) {

or am I misunderstanding something?

**Comment by StephenBrough** [comment]
Nope, you are correct. The constructor taking in a String has an @hide annotation in the source, so it's not available for use (without reflection). Anything marked with @hide will also not appear in the docs.
